tests/onnx/test_onnx_export_custom_module.py

```diff
@@ -22,7 +22,7 @@
 
 if is_torch_available():
     import torch
-    from transformers.models.deberta import modeling_deberta
+    from transformers.models.sew_d import modeling_sew_d
```

Comment:
`StableDropout` was removed from modeling_deberta in https://github.com/huggingface/transformers/pull/22105